apps/emqx_management/src/emqx_mgmt_api_clients.erl

```diff
@@ -1220,6 +1225,8 @@ list_client_msgs(MsgType, ClientID, QString) ->
             case emqx_mgmt:list_client_msgs(MsgType, ClientID, PagerParams) of
                 {error, not_found} ->
                     {404, ?CLIENTID_NOT_FOUND};
+                {error, shutdown} ->
+                    {404, ?CLIENT_SHUTDOWN};
```

Comment:
I think 404 is suitable here, because if the client process is shutdown a few moments earlier, we would respond with 404 not found.